src/pages/Contact.jsx

```diff
@@ -29,8 +41,10 @@ function Contact() {
       <h3>Reach Out</h3>
       <p>
         We are happy to answer any of your questions. You can email us{' '}
-        <a href="mailto:undocuscholar@googlegroups.com">here</a> and we&apos;ll
-        be in touch as soon as possible.
+        <a href="mailto:dreamscholars-contact@googlegroups.com?subject=Feature&body=">
```

Comment:
`Feature` is probably not the correct subject here. Also, as suggested elsewhere I suggest using `queryString.stringify()`. Maybe what this means is creating a helper library function to generate the `mailto:` URL. E.g. 

```jsx
        <a href={genMailToLink({subject: "General question"})}>
```

Where `getMailToLink` looks something like this in a TypeScript file:

```ts
interface MailToParams {
  bcc?: string,
  body?: string,
  cc?: string,
  subject?: string,
}

const teamEmail = "dreamscholars-contact@googlegroups.com";
const genMailToLink = (params: MailToParams) => `mailto:${teamEmail}?${queryString.stringify(params)}`;
```